src/tcgwars/logic/impl/gen8/BlueSkyStream.groovy

```diff
@@ -363,9 +381,13 @@ public enum BlueSkyStream implements LogicCardInfo {
         move "Victory Dive", {
           text "30 damage. You may search your deck for up to 2 cards and put them into your hand. Then, shuffle your deck."
           energyCost C, C
-          attackRequirement {}
           onAttack {
             damage 30
+            if (my.deck && confirm(text)) {
```

Comment:
I personally don't like to use confirm(text) for damaging attacks as it could give the impression that selecting no would result in the attack not doing damage.

Reply:
I'm 99% sure you aren't allowed to look at your deck and choose nothing. You either choose to check and get at least 1 card, or you choose not to check.

Reply:
> When I use Togekiss VMAX's move "Die Glide" and look at my deck, can I not select any cards from my deck?
No, you can't.
When you look at the deck, you must choose at least one card to add to your hand.

Reply:
You are correct about that. 
```suggestion
            if (my.deck && confirm("Search your deck for up to 2 cards?")) {
```
Is more what I had in mind. While it is an extremely minor tweak having the damage portion of the attack in the confirm message could create ambiguity as to whether selecting no would result in the whole attack being canceled or just the optional portion of it. I'm fine to leave this as is but wanted to clarify my comment.

Reply:
Oh, fair enough, you can go ahead and commit that suggestion.